java/app/src/main/java/org/vss/impl/postgres/PostgresBackendImpl.java

```diff
@@ -47,6 +47,20 @@ public PostgresBackendImpl(DSLContext context) {
     this.context = context;
   }
 
+  @Override
+  public void checkHealth() {
+    OffsetDateTime yesterday = OffsetDateTime.now().minusDays(1);
```

Comment:
```suggestion
    OffsetDateTime yesterday = OffsetDateTime.now().minusDays(2);
```

I think unfortunately this is needed since the records have no time element